tier4_planning_msgs/msg/SafetyFactorArray.msg

```diff
@@ -0,0 +1,4 @@
+std_msgs/Header header
```

Comment:
As with https://github.com/tier4/tier4_autoware_msgs/pull/155/files#r1879548845, do the predicted_paths and points of all safety factors share the same time and frame?

Reply:
> Since I don't suppose to aggregate PlanningFactorArray from different modules, all PlanningFactors are calculated in same planing cycle. So, I think it's not problem.

Save as above comment, all SafetyFactors are calculated in same frame.